site/data/authors.json

```diff
@@ -9,5 +9,6 @@
   "jroweboy": { "key": "jroweboy", "name": "jroweboy", "avatar": "https://community.citra-emu.org/user_avatar/community.citra-emu.org/jroweboy/120/30_1.png" },
   "jmc47": { "key": "jmc47", "name": "JMC47", "avatar": "https://community.citra-emu.org/user_avatar/community.citra-emu.org/jmc47/45/1487_1.png" },
   "anodium": { "key": "anodium", "name": "anodium", "avatar": "https://community.citra-emu.org/user_avatar/community.citra-emu.org/anodium/120/485_1.png"},
-  "saphiresurf": {"key": "saphiresurf", "name": "saphiresurf", "avatar": "https://community.citra-emu.org/user_avatar/community.citra-emu.org/saphiresurf/120/695_1.png"}
+  "saphiresurf": {"key": "saphiresurf", "name": "saphiresurf", "avatar": "https://community.citra-emu.org/user_avatar/community.citra-emu.org/saphiresurf/120/695_1.png"},
+  "CaptV0rt3x": {"key": "CaptV0rt3x", "name": "CaptV0rt3x", "avatar": "https://community.citra-emu.org/user_avatar/community.citra-emu.org/captv0rt3x/120/98_1.png"}
```

Comment:
Would you please add a comma after the last entry so that when you add an author next time, the line won't be changed? (That's what we usually do, and I don't know if it's any different for a JSON)

Reply:
JSON disallows trailing commas.

Reply:
Oh, then it's ok.